src/datasources/jwt/configuration/__tests__/jwt.configuration.ts

```diff
@@ -0,0 +1,7 @@
+import { registerAs } from '@nestjs/config';
+import { faker } from '@faker-js/faker';
+
+export default registerAs('jwt', () => ({
```

Comment:
Nit: we could create an `interface` from the "actual" `registerAs` to ensure that the signature or the `ReturnType` is the same.

Reply:
[fdb2d39](https://github.com/safe-global/safe-client-gateway/pull/1405/commits/fdb2d39d8e46451b3ef48f8c3c712e05fc1b6db2)